app/uk/gov/hmrc/hec/models/HECTaxCheckMatchResult.scala

```diff
@@ -16,24 +16,30 @@
 
 package uk.gov.hmrc.hec.models
 
-import julienrf.json.derived
-import play.api.libs.json.OFormat
-
 import java.time.ZonedDateTime
 
-sealed trait HECTaxCheckMatchResult
-
-object HECTaxCheckMatchResult {
+import ai.x.play.json.Jsonx
+import ai.x.play.json.SingletonEncoder.simpleName
+import ai.x.play.json.implicits.formatSingleton
+import play.api.libs.json.{Format, Json, OFormat}
 
-  final case class NoMatch(matchRequest: HECTaxCheckMatchRequest, dateTimeChecked: ZonedDateTime)
-      extends HECTaxCheckMatchResult
+sealed trait HECTaxCheckStatus extends Product with Serializable
```

Comment:
we may as well stick this in a separate file as well like in the LB PR - or at least the two repo's should follow the same approach so we don't have inconsistencies.

Reply:
updated in latest commits

Reply:
thanks :slightly_smiling_face: